.circleci/config.yml

```diff
@@ -361,22 +364,25 @@ commands:
           path: cypress/videos
       - store_artifacts:
           path: cypress/screenshots
+  install_cypress_dependencies:
+    steps:
+      - run:
+          name: Install cypress dependencies
+          command: sudo apt install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb -yq
```

Comment:
think `apt` recommends `apt-get` for scripting/non-interactive use 
https://unix.stackexchange.com/questions/590699/should-i-use-apt-or-apt-get-in-shell-scripting